fetchy/extractors/vaccinationDataPerDhbExtrator.ts

```diff
@@ -8,7 +8,7 @@ export const getVaccinationDataPerDhb = (
   rawHtml: string
 ): DhbVaccineDoseData[] => {
   const vaccinationTo90PerDhbTableHtml =
-    /Vaccinations\sto\s90%\sby\sDHB\<\/h3\>.*?\<tbody\>(.*?)\<\/tbody\>/gs.exec(
+    /Vaccinations\suptake\sto\s90%\sby\sDHB\<\/h3\>.*?\<tbody\>(.*?)\<\/tbody\>/gs.exec(
```

Comment:
This might be more precise?

```suggestion
    /\<a id="90pct" name="90pct"\>\<\/a\>.*?\<\/h3\>.*?\<tbody\>(.*?)\<\/tbody\>/gs.exec(
```

Reply:
Yeah you are right didn't see this id on it